json/src/main/java/org/teavm/flavour/json/tree/Node.java

```diff
@@ -32,9 +32,12 @@
     @JSBody(params = { "node" }, script = "return typeof node == 'string';")
     static native boolean isString(Node node);
 
-    @JSBody(params = { "node" }, script = "return node === null;")
+    @JSBody(params = { "node" }, script = "return node === null || node === undefined;")
```

Comment:
Please, remove `node === undefined` check from this method, that's incorrect

Reply:
OK, but now I had to add a check to not run the serializer for nullable values on the missing value